packages/push-platform-notifications-controller/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "../../tsconfig.packages.build.json",
+  "compilerOptions": {
+    "baseUrl": "./"
+  },
+  "references": [
+    {
+      "path": "../base-controller/tsconfig.build.json"
+    }
```

Comment:
Based on `package.json`, should we add `notifications-controller` and `profile-sync-controller`?

``` suggestion
    {
      "path": "../notifications-controller"
    },
    {
      "path": "../profile-sync-controller"
    },
```